improver_tests/psychrometric_calculations/test_PhaseChangeLevel.py

```diff
@@ -908,11 +908,13 @@ def test_too_many_cubes(self):
                 )
             )
 
-    def test_empty_cube_list(self):
+    def test_too_few_cubes(self):
```

Comment:
no cube error handling by as_cubelist function, not this class (i.e. this checks extraction logic of this cubelist by this class).